src/analysis/utils/cost-utils.test.ts

```diff
@@ -256,7 +256,7 @@ describe('GCP getCostDisplayForTool', () => {
 
   it('Will get compute cost and compute status for a running Jupyter runtime', () => {
     // Arrange
-    const expectedResult = 'Running $0.05/hr';
+    const expectedResult = 'Running $0.06/hr';
```

Comment:
<img width="654" alt="image" src="https://github.com/DataBiosphere/terra-ui/assets/85642387/9c600b2e-3415-40f6-9a7e-bd8ac7e83e3a">
Small thing, but I'm not seeing the price increase reflected on the creation page here